docs/data/toolpad/core/components/sign-in-page/BrandingSignInPage.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { AppProvider, SignInPage } from '@toolpad/core';
+import { createTheme } from '@mui/material/styles';
+import { useColorSchemeShim } from 'docs/src/modules/components/ThemeContext';
+import { getDesignTokens } from './brandingTheme';
+
+const providers = [
+  { id: 'github', name: 'GitHub' },
+  { id: 'google', name: 'Google' },
+  { id: 'credentials', name: 'Email and Password' },
+];
+
+const BRANDING = {
+  logo: (
+    <img
+      src="https://mui.com/static/logo.svg"
+      alt="MUI logo"
+      style={{ height: 24 }}
+    />
+  ),
+  title: 'MUI',
+};
+
+export default function BrandingSignInPage() {
+  const { mode, systemMode } = useColorSchemeShim();
+  const calculatedMode = (mode === 'system' ? systemMode : mode) ?? 'light';
+  const brandingDesignTokens = getDesignTokens(calculatedMode);
+  const THEME = createTheme({
```

Comment:
Why can't the default theme just look good here out of the box?

Reply:
Do you mean not adding a custom theme in this demo? I was trying to demonstrate how the sign in page could be styled completely differently with a custom theme in this demo.

Reply:
Ah got it, I'm already going to be exemplifying theming in a very basic way in the `AppProvider` component docs page, but I guess it might make sense to have something more specific to show how it would look in the sign up page, so I guess this should be cool.